vxs.inc

```diff
@@ -5,7 +5,7 @@
 #  define VXS_CLASS "version"
 #  define VXSp(name) XS_##name
 /* VXSXSDP = XSUB Details Proto */
-#  define VXSXSDP(x) x
+#  define VXSXSDP(x) x, 0
```

Comment:
This looks problematic given the file's header. I think it expects to retain back compat, which may be hard with the current approach.

Reply:
This is only used when version is part of core, it can only be problem if we backport version to maint-older, which we don't typically.